gui/functionality/idact_notebook.py

```diff
@@ -6,15 +6,28 @@
 class IdactNotebook:
     def __init__(self, idact_app):
         idact_app.ui.deploy_button.clicked.connect(idact_app.deploy_notebook)
+        idact_app.ui.cluster_name_deployn_edit.setText(idact_app.parameters['deploy_notebook_arguments']['cluster_name'])
+        idact_app.ui.nodes_edit.setValue(idact_app.parameters['deploy_notebook_arguments']['nodes'])
+        idact_app.ui.cores_edit.setValue(idact_app.parameters['deploy_notebook_arguments']['cores'])
+        idact_app.ui.memory_edit.setValue(idact_app.parameters['deploy_notebook_arguments']['memory_value'])
+        idact_app.ui.memory_unit_box.setCurrentText(idact_app.parameters['deploy_notebook_arguments']['memory_unit'])
+        idact_app.ui.walltime_edit.setText(idact_app.parameters['deploy_notebook_arguments']['walltime'])
 
     @addToClass(IdactApp)
     def deploy_notebook(self):
         cluster_name = self.ui.cluster_name_deployn_edit.text()
+        self.parameters['deploy_notebook_arguments']['cluster_name'] = cluster_name
         nodes = int(self.ui.nodes_edit.text())
+        self.parameters['deploy_notebook_arguments']['nodes'] = nodes
         cores = int(self.ui.cores_edit.text())
-        memory = self.ui.memory_edit.text()
```

Comment:
Still think it should stay as string - we are only using it as string no need to cast it into int only to cast it back.

Reply:
I changed that but I needed to convert memory to int while passing it as the argument to the 'setValue' method because it requires int as the argument